.travis.yml

```diff
@@ -9,5 +9,6 @@ install:
   - go get github.com/robertkrimen/otto
   - go get github.com/howeyc/fsnotify
   - go get github.com/robfig/gettext/po
+  - go get github.com/robfig/soy
```

Comment:
why do we need this?

Reply:
the imports are in the form github.com/robfig/soy/{package}

Reply:
Hmm, yeah.  Not sure Travis will work for you as most of the code uses robfig/soy/ and not ovkulkarni/soy, right?  Like you aren't testing just the code in your repo.